NosTests/GiftWrapperTests.swift

```diff
@@ -0,0 +1,45 @@
+import XCTest
+
+final class GiftWrapperTests: XCTestCase {
+    func testDirectMessageWrapper() throws {
+        let senderKeyPair = KeyFixture.alice
+        let receiverKeyPair = KeyFixture.bob
+        
+        let wrappedDM = try DirectMessageWrapper.wrap(
+            message: "Are you going to the party tonight? 🎉",
+            senderKeyPair: senderKeyPair,
+            receiverPubkey: receiverKeyPair.publicKeyHex
+        )
+        
+        XCTAssertEqual(wrappedDM.kind, EventKind.giftWrap.rawValue)
+        XCTAssertEqual(wrappedDM.tags, [["p", receiverKeyPair.publicKeyHex]])
+        XCTAssertGreaterThan(
+            Date(timeIntervalSince1970: Double(wrappedDM.createdAt)),
+            Date().addingTimeInterval(-TWODAYS)
+        )
+        XCTAssertLessThan(Date(timeIntervalSince1970: Double(wrappedDM.createdAt)), Date.now)
+        XCTAssertNotEqual(wrappedDM.pubKey, senderKeyPair.publicKeyHex)
+        XCTAssertNotEqual(wrappedDM.pubKey, receiverKeyPair.publicKeyHex)
+        XCTAssertFalse(wrappedDM.content.contains("party"))
```

Comment:
I would like to see more tests for the `wrappedDM.content` - that it is a valid seal, that the seal contains the correct rumor, etc. Maybe these should be unit tests on GiftWrapper.swift instead of integration tests here, either or both are fine with me.

Also if you want to skip writing these for time's sake then that's ok given the current deadlines the team has. In that case let's create a ticket to circle back and write them later.

Reply:
I added this to https://github.com/planetary-social/nos/issues/1076